weblab/datasets/views.py

```diff
@@ -204,7 +204,7 @@ def get(self, request, *args, **kwargs):
 
         content_type, _ = mimetypes.guess_type(filename)
         if content_type is None:
-            content_type = 'application/octet-stream'
+            content_type = 'text/csv'
```

Comment:
Not convinced by this! What files aren't being recognised as CSV and should be?

Reply:
without it, if I put the path to the CSV for building graphs in my browser (to check it out), the browser would not recognise it as CSV and try to download it as a file, instead of displaying it as nicely formatted CSV I could inspect.

Reply:
Can you delve into why this is, rather than pretending all unrecognised names are CSV? What is the filename that isn't being recognised? Is it a typical Web Lab pattern, in which case use https://docs.python.org/3/library/mimetypes.html#mimetypes.add_type to teach the library?

Reply:
seems to be a known bug with the python version we're running https://bugs.python.org/issue13952